### ISSUE TITLE
[CoE Starter Kit - QUESTION] QUESTION

### ISSUE THREAD

**Issue by DominicMeznik** [microsoft/coe-starter-kit#5699]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

I´m currently upgrading the starter kit to the newest version (May release) and ran into an issue, while trying to import the core solution.

Trying to import the solution always returns following error: 

Code: BadRequest Content: DataflowId already exists under a different environment

I am using 2 different Environments (TST & PRD) so it totally makes sense that it would throw this error for TST, since I initially installed the kit on the production environment, where these dataflows exist.

Do you have any suggestions how I could fix this, so I can import and test changes on TST?

(Note: I am not using the Datalake implementation)

Thank you!

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.8

### What app or flow are you having the issue with?

Core Solution import

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team

**Comment by DominicMeznik** [comment]
@Jenefer-Monroe Thank you! That actually worked, but I had to use XRM Toolbox since no data was displayed in Excel.

I also analyzed the issue and found that it most likely came from me duplicating the Environment instead of creating a new one from scratch. I guess this could be added to the documentation.

I now also have to delete all the solutions after importing the updates, as the system doesn´t recognize the import as an Upgrade.

I guess that´s on me being lazy :P
